### ISSUE TITLE
Add more arguments for Cosine_with_restarts scheduler

### ISSUE THREAD

**Issue by Azurlifrfa** [kohya-ss/sd-scripts#73]
Please add some parameters in your finetuner, like Minimum and maximum learning rate, rate of increasing/decreasing learning rate per cycle, numbers of epochs per cycle and warmup steps per cycle, like in Aria1th's monkey patch          
>https://github.com/aria1th/Hypernetwork-MonkeyPatch-Extension
